esti/main_test.go

```diff
@@ -70,11 +69,11 @@ func TestMain(m *testing.M) {
 		Name:      "esti",
 		StorageNS: "esti-system-testing",
 	}
+
 	if *useLocalCredentials {
-		params.AdminAccessKeyID = "AKIAIOSFDNN7EXAMPLEQ"
-		params.AdminSecretAccessKey = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+		params.AdminAccessKeyID = *flag.String("admin-access-key-id", "AKIAIOSFDNN7EXAMPLEQ", "lakeFS Admin access key ID")
+		params.AdminSecretAccessKey = *flag.String("admin-secret-access-key", "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY", "lakeFS Admin secret access key")
```

Comment:
Usually the assign-flag-to-variable form (l. 55-56 above) is easier to read.  One advantage is that all the flags appear together.